src/pyOpenMS/tests/unittests/test_MSExperiment_rasterize2D.py

```
@@ -0,0 +1,265 @@
+    
+    with pytest.raises(ValueError, match="Invalid aggregation mode"):
+        exp.rasterize2D(output, 100.0, 200.0, 500.0, 600.0, 1, "invalid")
+
+
+def test_rasterize2D_exception_inverted_rt_range():
+    """Test that inverted RT range raises exception."""
+    exp = create_test_experiment()
+    output = np.empty((10, 10), dtype=np.float32)
+    
+    # Equal RT values
+    with pytest.raises(Exception):
```

Ruff (0.14.14)

[warning] Do not assert blind exception: `Exception`

(B017)